packages/admin/theme/utility/appearance.ts

```diff
@@ -52,6 +52,16 @@ export default css`
     ${aBaseInput()}
   }
 
+  .a-pagination-input {
+    max-width: 30px;
+    -moz-appearance: textfield;
+    text-align: left;
+    &:hover,
+    &:focus {
+      -moz-appearance: auto;
+    }
+  }
+
```

Comment:
@jen-castiron Does this look okay? I'm definitely still learning the details of styling forms.

Reply:
Hmm... the linter doesn't like the `-moz` prefixes, but I think they're needed here?

Reply:
I don't think the bug here is the numeric controls (which are good for accessibility), just the width. (Check with Zach to be sure.) If the issue is the numeric controls, then you'd want to use just `appearance: textfield` without the mod prefix.